### ISSUE TITLE
Intangible Integration

### ISSUE THREAD

**Issue by Thiana** [legendblade/CraftingHarmonics#38]
Could you please add integration to modify the various forms of crafting in Intangible? Thank you.

PS: This is not possible right now since Emoniph is still porting it from 1.8.9 to 1.9 (http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2567717-intangible-0-0-25-alpha?page=79#c1571). I just wanted the request recorded :)

**Comment by legendblade** [comment]
Yeah, I looked at the title and my immediate reaction was "Intangible was released for 1.9?!", and then I got sad, because I saw it was still in progress...

I'll definitely see what I can do when he gets a 1.9 release out. :)
